rusk/src/lib/http/chain/graphql/data.rs

```diff
@@ -8,6 +8,22 @@ use std::ops::Deref;
 
 use async_graphql::{FieldError, FieldResult, Object, SimpleObject};
 use node::database::{Ledger, LightBlock, DB};
+use serde::{Deserialize, Serialize};
+
+/// Pair of (block height, block hash) of the last block and the last finalized
+/// block.
+#[derive(Serialize, Deserialize)]
+pub struct BlockPair {
+    pub last_block: (u64, String),
```

Comment:
The sub fields should also be documented right? Clippy should throw and error if the fields are public and not documented